Projects/Market/Sources/MarketStore.swift

```diff
@@ -2,28 +2,41 @@ import Flow
 import Foundation
 import Presentation
 import hCore
+import hGraphQL
+import Apollo
 
 public struct MarketState: StateProtocol {
     var market: Market = .sweden
+    var onboardingIdentifier: String? = nil
 
     public init() {}
 }
 
 public enum MarketAction: ActionProtocol {
     case selectMarket(market: Market)
+    case setOnboardingIdentifier(id: String)
 }
 
 public final class MarketStore: StateStore<MarketState, MarketAction> {
+    @Inject var client: ApolloClient
+    @Inject var store: ApolloStore
+    
     public override func effects(
         _ getState: @escaping () -> MarketState,
         _ action: MarketAction
     ) -> FiniteSignal<MarketAction>? {
         switch action {
         case let .selectMarket(market):
             Localization.Locale.currentLocale = market.preferredLanguage
+            return self.client
+                .perform(
+                    mutation: GraphQL.CreateOnboardingQuoteCartMutation(input: .init(market: market.graphQL, locale: market.preferredLanguage.rawValue))
```

Comment:
Doesn't this Query make more sense in `Offer`?

Reply:
I'm going to start implementing this now on Android. Popping in here to bounce som ideas. 

We need to make this query before creating a quote in embark, this is the mutation specified in angel:
```
mutation CreateSwedishQuote(
    $swedishHomeQuoteId: ID!
    ...
)
```

Can we use `EmbarkStoryType` to know when we need to generate a quote cart id? We could then check `EmbarkStoryType.APP_ONBOARDING_QUOTE_CART`. We would then need to add more types for moving flow etc. But maybe thats ok? 

This could perhaps be done at the same time we are querying the story.

Thoughts?

Reply:
@sampettersson we need the `quoteCartID` in Embark unfortunately. I was talking with robin about adding this into embark as an `apiFragment`.

Reply:
@sampettersson I will move this query to embark. Do we want to tuck quote cart behind a feature flag? @hugokallstrom